src/components/DpDataTable/DpDataTable.vue

```diff
@@ -568,6 +614,12 @@ export default {
       }, {})
 
       this.allWrapped = status
+    },
+
+    updateSessionStorage (storageName, value) {
```

Comment:
`storageName` is not set anywhere.

Reply:
what do you mean? It is an argument in the func, should it be set somewhere?

Reply:
my bad, did not look closely enough :eyeglasses: